data/templates/gandalf/config.yml

```diff
@@ -0,0 +1,9 @@
+name: Confused Gandalf
+link: http://knowyourmeme.com/memes/confused-gandalf
+default:
+- ''
+- What did I just say?
+aliases:
+- gandalf
```

Comment:
This line should. be removed as it's already the default key for this template based on the directory name.